mu-plugins/central-hub/tests/phpunit/integration/config-store/loadConfigFromFilesystem.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ *  Tests for external API function loadConfigFromFilesystem()
+ *
+ * @package    spiralWebDb\centralHub\Tests\Integration\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\ConfigStore;
+
+use Brain\Monkey;
+use function KnowTheCode\ConfigStore\_the_store;
+use function KnowTheCode\ConfigStore\loadConfigFromFilesystem;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+
+/**
+ * Class Tests_LoadConfigFromFilesystem
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\ConfigStore
+ * @group   config-store
+ */
+class Tests_LoadConfigFromFilesystem extends Test_Case {
+
+	/**
+	 * Test loadConfigFromFilesystem() should merge defaults with config and return a store key.
+	 */
+	public function test_should_merge_defaults_and_return_store_key() {
+		$path_to_file  = CENTRAL_HUB_ROOT_DIR . '/tests/phpunit/fixtures/test-cpt-config.php';
+		$defaults      = [
+			'aaa' => 37,
+			'eee' => 'Coding is fun!',
+		];
+		$merged_config = [
+			'aaa' => 'bbb',
+			'ccc' => 'ddd',
+			'eee' => 'Coding is fun!',
+		];
+
+		$this->assertSame( 'foo', loadConfigFromFilesystem( $path_to_file, $defaults ) );
+		$this->assertEquals( $merged_config, _the_store( 'foo' ) );
```

Comment:
@rgadon107 Notice how we are checking not only the returned store key, but also that this function actually did do the steps we expect, i.e.:

1. Loaded from the file.
2. Merged with the defaults.
3. Stored the merged config into the store.
4. Returns the store key.

By checking what gets stored in the store, we are validating the expected behavior.